main.tf

```diff
@@ -1,9 +1,65 @@
 terraform {
-  required_version = ">= 1.3.0"
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.0"
+      version = "~> 5.0"
     }
+    cloudflare = {
+      source  = "cloudflare/cloudflare"
+      version = "~> 5.0"
+    }
+    null = {
+      source  = "hashicorp/null"
+      version = "~> 3.0"
+    }
+  }
+
+
+  required_version = ">= 1.11"
```

Comment:
is 1.11 required? I dont think we us anything that requires 1.11. 1.3.2 is pretty safe.